src/network/netmon_mgmt.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include "sntp_client.h"
+#include "dhcpwait.h"
+#include "netmon_stats.h"
+
+LOG_MODULE_REGISTER(netmon_mgmt, CONFIG_LOG_DEFAULT_LEVEL);
+
+#define NETMON_THREAD_PRIORITY 5
+static struct k_work_delayable time_sync;
+
+/* Network monitor management worker thread work item to sync
+ * the time with SNTP server.
+ */
+static void netmon_mgmt_time_sync(struct k_work *work)
+{
+
```

Comment:
Extra line ... if you haven't, please run these through `clang-format` with Zephyr's config file for that, although maybe it just doesn't catch the leading space issues.